xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -229,9 +229,11 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.String&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of command-line arguments to use when starting the application.</summary>
+        <value>A collection of string values.</value>
```

Comment:
We usually explain what the return means and not what the type is. Most of the time this is merely a duplication of what the summary says (and /// comments don't even support this for properties - can give you context if you want)